src/Config.php

```diff
@@ -74,7 +74,8 @@ protected function configureOptions(OptionsResolver $resolver)
             'constructorParamsDefaultToNull' => false,
             'soapClientClass'               => '\SoapClient',
             'soapClientOptions'             => array(),
-            'proxy'                         => false
+            'proxy'                         => false,
+            'bracketedArrays'               => false
```

Comment:
I appreciate the intent here but I would prefer not to add the configuration option and just use PHP 5.4 short array going forward and remove PHP 5.3 support entirely (including from the Travis build matrix). This way we avoid adding unneeded complexity. PHP 5.3 was EOL over two years ago.